package.json

```diff
@@ -41,6 +42,9 @@
     "rimraf": "2.2.8"
   },
   "scripts": {
+    "build": "npm run build:client && npm run build:server",
+    "build:client": "cd client && brunch b",
```

Comment:
Can we merge that one? The goal of this PR is to make Files available on NPM. Making build proper is another subject. It's better if we separate that two jobs.